pkg/csource/options.go

```diff
@@ -38,6 +38,7 @@ type Options struct {
 	EnableBinfmtMisc bool `json:"binfmt_misc,omitempty"`
 	EnableCloseFds   bool `json:"close_fds"`
 	EnableKCSAN      bool `json:"kcsan,omitempty"`
+	EnableDevlinkPCI bool `json:"devlinkpci,omitempty"`
```

Comment:
This also needs to be enabled in DefaultOpts function.

Reply:
ok, done.